site/app/templates/forum/ShowForumThreads.twig

```diff
@@ -14,6 +14,12 @@
         $("form").areYouSure();
         loadThreadHandler();
         initSocketClient();
+
+        //delete new_thread Class to mark posts as seen in frontend for users who access posts via href or url
+        thread_id = window.location.pathname.substring(window.location.pathname.lastIndexOf('/') + 1);
```

Comment:
Thanks for the PR @HeatMint ,

I think we want to avoid parsing the URL to get the current thread incase this changes in the future. As a suggestion you can access the current selected thread with the following jquery:

```js
$('data#current-thread').val()
```

Once you have the thread id you can just find the element by id using 

```js
$(`#thread_box_link_${thread_id} .thread_box`).removeClass('new_thread');
```